### ISSUE TITLE
runtime error: slice bounds out of range [:-1]

### ISSUE THREAD

**Issue by Dialga** [proofrock/ws4sqlite#11]
My database name is `db` which causes this error.

```
$ ./ws4sqlite -db db -serve-dir .
ws4sqlite 0.12.3
- Based on SQLite v3.39.4
panic: runtime error: slice bounds out of range [:-1]

goroutine 1 [running]:
main.parseCLI()
        /home/mano/ws4sqlite/src/cli.go:87 +0xdf9
main.main()
        /home/mano/ws4sqlite/src/ws4sqlite.go:67 +0x11b
```

**Comment by Dialga** [comment]
Also, I'm unsure why your home path is hardcoded in the error message.

**Comment by proofrock** [comment]
> My database name is `db` which causes this error.

Oh my. Thanks. Will look into it as soon as I get a reliable internet connection.

**Comment by proofrock** [comment]
> Also, I'm unsure why your home path is hardcoded in the error message.

I need to use `-trimpath` it seems. Would you care to open another issue?

**Comment by Dialga** [comment]
> > Also, I'm unsure why your home path is hardcoded in the error message.
> 
> I need to use `-trimpath` it seems. Would you care to open another issue?

Done.

**Comment by proofrock** [comment]
> Oh my. Thanks. Will look into it as soon as I get a reliable internet connection.

This looks like to be broken since #4, the code was meant to strip the extension but it doesn't do that, it just strips the last three characters (they used to be fixed to `.db`). Actually, currently the YAML file name is broken too, when using an extension that is not two letters long. Thanks for pointing out this oversight. Fixing it could be not backwards-compatible for the reasons above, but it was wrong anyway, so this is a moot point.

**Comment by proofrock** [comment]
Should be fixed in the develop branch. Would you be so kind and test it? If you can't compile the branch, and you still want to test it, let me know your arch and I'll compile it and send it to you. Thanks!

**Comment by proofrock** [comment]
Released in v0.12.4
